BeanHub.API/src/main/java/com/bean/api/util/RecipeAverageRating.java

```diff
@@ -0,0 +1,27 @@
+package com.bean.api.util;
+
```

Comment:
Is this class used as a request param for a request?
If so can you please move this to the requests folder